translate/storage/test_php.py

```diff
@@ -484,57 +484,78 @@ def test_parsing_arrays_declared_in_a_single_line(self):
 
     def test_parsing_arrays_with_no_keys(self):
         """Parse an array with no keys."""
-        phpsource = """$days = array('Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday');"""
+        phpsource = """<?php
+$days = array(
+    'Sunday',
+    'Monday',
+    'Tuesday',
+    'Wednesday',
+    'Thursday',
+    'Friday',
+    "Saturday",
+);
```

Comment:
Doesn't it work if it is kept in a single line? I am asking because that is how it was found.

Reply:
Parsing works just fine, but serializing formats it this way, that's why I've changed it.

Reply:
Ok, I guess we will have to allow this to happen. On the other PHP code should be like this, so I guess it is not that bad.